src/mots/utils.py

```diff
@@ -39,3 +38,32 @@ def parse_real_name(real_name):
         return match.groupdict()
     else:
         return {"name": None, "info": None}
+
+
+class Disk:
+    """Utilities to read and write settings files and directories to disk."""
+
+    def __init__(self):
+        self.home = Path.home()
+        self.resource_directory = self.home / ".mots"
+        self.overrides_file = self.resource_directory / "settings.yaml"
```

Comment:
These values seem like they should be able to be set via the CLI/env like the others. Unless I'm missing something it seems like we use them before we create and load the `Settings` object. It's also a little odd to see details that are specific to the `Settings` class leaked into this module. Maybe this `Disk` object should be merged into `Settings`? They seem coupled and the docstring mentions the class is specifically for writing the settings out.

Reply:
There are reasons why this class is in a separate module (to keep definitions cleaner in `settings.py`, for example, as this class is used after it is loaded but before the `Settings` class is defined). These values also shouldn't be changed, rather they are discovered based on the home directory (which is a fixed value).

Having said that, I don't particularly like my implementation here and I think there's clean up/relocation that can be done here, so I will take a stab at that.